content/chainguard/chainctl-usage/chainctl-images.md

```diff
@@ -11,21 +11,64 @@ images: []
 weight: 070
 ---
 
-This page presents some of the more commonly used `chainctl images` commands. For a full reference of all commands with details and switches, see [chainctl Reference](/chainguard/chainctl/).
+This page presents some of the more common ways to learn about the Chainguard Containers that are available to you. We use `chainctl images` commands to list available repositories and container images, examine images more closely, and compare them to one another.
```

Comment:
I know the product name has changed, but in some contexts I feel it can be confusing; in my first pass on this sentence I imagined a command that worked like `docker ps`. I would probably use "Chainguard Container Images" in this spot specifically to make it clearer that we're talking about images.

Reply:
I dont think we should introduce a new title-cased product name like "Chainguard Container Images" .. I think the proposed change is good.

Reply:
Thank you. I think both of you are right. Much appreciated!